hooks/data.js

```diff
@@ -8,19 +8,27 @@ var _           = require('lodash');
 /** Entity for tracking hooks and associated state **/
 var Hook = base.Entity.configure({
   version:              1,
-  partitionKey:         base.Entity.keys.StringKey('groupId'),
+  partitionKey:         base.Entity.keys.StringKey('hookGroupId'),
   rowKey:               base.Entity.keys.StringKey('hookId'),
   properties:           {
-    groupId:            base.Entity.types.String,
+    hookGroupId:        base.Entity.types.String,
     hookId:             base.Entity.types.String,
     metadata:           base.Entity.types.JSON,
+    // task template
     task:               base.Entity.types.JSON,
+    // pulse bindings
     bindings:           base.Entity.types.JSON,
+    // timings for the task (in fromNow format, e.g., "1 day")
     deadline:           base.Entity.types.String,
     expires:            base.Entity.types.String,
+    // schedule for this task
```

Comment:
I think this is a good place to document the form of the JSON object... at least if it's not trivial..
Or just say what sub-schema it follows... ie. some property of some schema...